docs/source/theming/create-theme-add-on.md

```diff
@@ -54,8 +54,28 @@ This approach is best when you want to reuse the same theme across multiple proj
 
 For both of the foregoing methods to create a Volto theme add-on, you'll next need to declare the active theme.
 
-You can declare the active theme in either of the two following ways, the first of which is preferred.
+You can declare the active theme in either of the three following ways.
+They are listed here in order of precedence, from highest to lowest.
+The last one has the highest precedence and is the one that prevails.
```

Comment:
It seems there's a mistake here. It appears that `volto.config.js` takes precedence over the add-ons, but it's listed in position 2 while the add-ons are in position 1.